ui/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "moody",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:3000",
+  "proxy": "https://s2-reader.apps.allenai.org",
```

Comment:
What's the reason for this change? Will this impact our ability to test the dev version of the UI with the dev version of the API?

Reply:
Ah, this just makes things work "out of the box".  

If a user wants to run against a local API they simply need to edit this value (and remember not to commit it).

That said I think you're right that it's a bit kludgy as is. Let me see if I can make this a bit easier.  Ideally the default would be this (so that it works), but it'd be possible for a user to set an environment variable or config bit to override this for local execution (i.e. so they don't have to worry about not committing the effect).

Reply:
If there's an easy solution with environment variables then yeah, that sounds like a nice convenience. Otherwise, I think this works, especially if you added a note to the README indicating what to change if you're developing with a dev version of the API.

Reply:
It was super easy. Thanks for the nudge :).

Here's the docs that explain the mechanics:
https://github.com/allenai/scholar-reader/blob/444c8b2ef2d68f90c65260dfea348f1362272397/ui/README.md#using-a-local-api